### DISCUSSION TITLE
Trading one coin for another - spot price

### DISCUSSION THREAD

**Discussion by jk-newbie** [eprbell/rp2#38] [General]
I'm working through the process of manually configuring my input files.  In the process I  am getting a warning that leads me to this question:      If one were to buy ETH using BTC, a taxable event, what value does one put in the 'spot price' column of the 'manual_in.csv file?   I would have thought it would be the price of BTC at the time of sale thus representing my basis.  But in doing so I get the warning "crypto_in * spot_price != fiat_in_no_fee".       And forcing spot_price to equivalent USD would likely still throw the same warning/error.   Please advise.

The larger question is how does one prepare the csv files for coin-to-coin trading?  Thanks

**Comment by eprbell** [comment]
Hi, are you using DaLI + RP2 or just RP2? RP2 conversions are covered in [the RP2 FAQ](https://github.com/eprbell/rp2/blob/main/docs/user_faq.md#how-to-handle-conversion-of-a-cryptocurrency-to-another). Not sure about the warning: can you provide transactions details (but be sure to [read this first](https://github.com/eprbell/rp2/blob/main/CONTRIBUTING.md#reporting-bugs) before posting any personal data)?

**Reply by jk-newbie** [reply]
Operator error on my part - I was looking in the DaLI docs, not the RP2 docs for how to manage coin-to-coin trades.  My compliments on what appears to be a very impressive non-trivial effort.  As a side note, I've been using the manual input method.  I appreciate that you've included & maintained this option, though I will say it has been tedious dealing with how different brokers book their fees and getting the entries correct.  Once again, thanks for this open source tool.

**Comment by eprbell** [comment]
Thanks! Really glad you found it useful. If you'd like to help the projects, please consider leaving a star on their Github page.
